spec/models/post_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.describe Post, type: :model do
+  let(:blank_values) { [nil, ""] }
+  it { should have_valid(:body).when("Any 1 .!@#$%^&*}text") }
+  it { should_not have_valid(:body).when(*blank_values,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.